### DISCUSSION TITLE
slow downloads ControlNET preprocessors

### DISCUSSION THREAD

**Discussion by inck86** [vladmandic/sdnext#460] [Q&A]
Downloading of each preprocessor for ControlNet takes place at a speed of 20-90 kb/s, while the Internet allows downloading 10Mb/s.
Maybe there is a way to download directly from another source? Has anyone else encountered this?

Tried to download from https://cloudstor.aarnet.edu.au/plus/s/lTIJF4vrvHCAI31/download directly, but the speed is the same, maybe their server is overloaded..

![image](https://user-images.githubusercontent.com/119116007/234098871-fef837ca-0bb8-40f7-a932-1a4cd067d67c.png)

**Comment by inck86** [comment]
through A1111 the same thing went with 5-7 Mb/s...
![image](https://user-images.githubusercontent.com/119116007/234107773-a0142d5f-b892-4a05-b63e-6494e1fcb3b3.png)

**Comment by vladmandic** [comment]
you do realize that its the same extension and extension is doing the download? the core has no impact on it.

**Reply by inck86** [reply]
> you do realize that its the same extension and extension is doing the download? the core has no impact on it.

I suspect that the cloudstore server is overloaded. I don't know what explains the difference in downloads, but I'm not making it up) although of course I logically understand that this is all very strange. with huggingface, by the way, preprocessors swing stably in Vlad automatic
